### ISSUE TITLE
Crashes on start: `safe-json-stringify` was removed from dependencies but is still used in `openapi3.js`

### ISSUE THREAD

**Issue by ForsakenHarmony** [Mermade/widdershins#317]
**Describe the bug**
After installing globally, the cli crashes on start

<details>
<summary>Error</summary>

```
❯ widdershins
internal/modules/cjs/loader.js:979
  throw err;
  ^

Error: Cannot find module 'safe-json-stringify'
Require stack:
- /home/hrmny/.local/share/npm/lib/node_modules/widdershins/lib/openapi3.js
- /home/hrmny/.local/share/npm/lib/node_modules/widdershins/lib/openapi2.js
- /home/hrmny/.local/share/npm/lib/node_modules/widdershins/lib/index.js
- /home/hrmny/.local/share/npm/lib/node_modules/widdershins/widdershins.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:976:15)
    at Function.Module._load (internal/modules/cjs/loader.js:859:27)
    at Module.require (internal/modules/cjs/loader.js:1036:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (/home/hrmny/.local/share/npm/lib/node_modules/widdershins/lib/openapi3.js:8:18)
    at Module._compile (internal/modules/cjs/loader.js:1147:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1167:10)
    at Module.load (internal/modules/cjs/loader.js:996:32)
    at Function.Module._load (internal/modules/cjs/loader.js:896:14)
    at Module.require (internal/modules/cjs/loader.js:1036:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/hrmny/.local/share/npm/lib/node_modules/widdershins/lib/openapi3.js',
    '/home/hrmny/.local/share/npm/lib/node_modules/widdershins/lib/openapi2.js',
    '/home/hrmny/.local/share/npm/lib/node_modules/widdershins/lib/index.js',
    '/home/hrmny/.local/share/npm/lib/node_modules/widdershins/widdershins.js'
  ]
}
```
</details>

**To Reproduce**
1. `> npm i -g widdershins`
2. `> widdershins`

**Expected behavior**
Does not crash

**Versions**:
- Node.js (note only LTS versions are supported): v13.12.0

**Additional context**
https://github.com/Mermade/widdershins/blob/master/lib/openapi3.js#L8

**Comment by MikeRalphson** [comment]
Still worked (and passed all tests) due to a transitive dependency. Unsure why installing globally would have resulted in different behaviour. Anyway, thanks for reporting this, fixed in `v4.0.1`.

**Comment by ForsakenHarmony** [comment]
Maybe a dev dependency :woman_shrugging:
